### ISSUE TITLE
Group Chat - List does not scroll to the top after selecting user manually

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#9742]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/docs/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
Break down in numbered steps

## Expected Result:
Describe what you think should've happened

## Actual Result:
Describe what actually happened

## Workaround:
Can the user still use Expensify without this being fixed? Have you informed them of the workaround?

## Platform:

Where is this issue occurring?

- Web
- iOS
- Android
- Desktop App
- Mobile Web

**Version Number:** 
**Reproducible in staging?:** 
**Reproducible in production?:** 
**Email or phone of affected tester (no customers):** 
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation
**Expensify/Expensify Issue URL:** 
**Issue reported by:** 
**Slack conversation:** 

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by Julesssss** [comment]
Based on the steps this seems valid to me, but please correct me if I missunderstand @roryabraham

**Comment by roryabraham** [comment]
Yep, this looks valid to me. @techievivek I can grab this and work on a fix since I've worked on this code a lot recently

**Comment by roryabraham** [comment]
Got a PR up, ran out of steam before I finished posting screenshots, etc... https://github.com/Expensify/App/pull/9798. Will finish it up tomorrow.

**Comment by roryabraham** [comment]
PR is in review
